2018-06-27 07:04:54.988 27577 DEBUG oslo_concurrency.lockutils [req-df7422fa-3f1e-421a-909d-4e64447645b4 c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Acquired semaphore "dhcp-agent-network-lock-fc20bd18-b90e-4028-a59e-82494c1cf687" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:04:54.989 27577 INFO neutron.agent.dhcp.agent [req-df7422fa-3f1e-421a-909d-4e64447645b4 c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T05:04:52Z, description=, device_id=50c3fd2e-0812-44c4-8b27-d8c497a7bd7e, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'90f939ae-63a5-46e2-b562-af2be2ccf784', u'ip_address': u'11.132.1.12'}], id=bffc516f-6853-4c5c-b6a9-05e09148c677, mac_address=fa:16:3e:27:10:1b, name=, network_id=fc20bd18-b90e-4028-a59e-82494c1cf687, port_security_enabled=True, project_id=0459e438b74d46959d6c89c9619846d9, revision_number=3, security_groups=[u'2b3abfdf-6a0f-454e-a746-e974e35a56de'], status=DOWN, tags=[], tenant_id=0459e438b74d46959d6c89c9619846d9, updated_at=2018-06-27T05:04:53Z
2018-06-27 07:04:54.991 27577 DEBUG neutron.agent.dhcp.agent [req-df7422fa-3f1e-421a-909d-4e64447645b4 c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Calling driver for network: fc20bd18-b90e-4028-a59e-82494c1cf687 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:04:54.993 27577 DEBUG neutron.agent.linux.dhcp [req-df7422fa-3f1e-421a-909d-4e64447645b4 c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Building host file: /var/lib/neutron/dhcp/fc20bd18-b90e-4028-a59e-82494c1cf687/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:04:54.996 27577 DEBUG neutron.agent.linux.dhcp [req-df7422fa-3f1e-421a-909d-4e64447645b4 c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Done building host file /var/lib/neutron/dhcp/fc20bd18-b90e-4028-a59e-82494c1cf687/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:04:55.003 27577 DEBUG neutron.agent.linux.utils [req-df7422fa-3f1e-421a-909d-4e64447645b4 c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '31895'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:04:55.027 27577 DEBUG neutron.agent.linux.dhcp [req-df7422fa-3f1e-421a-909d-4e64447645b4 c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Reloading allocations for network: fc20bd18-b90e-4028-a59e-82494c1cf687 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:04:55.028 27577 DEBUG neutron.agent.linux.utils [req-df7422fa-3f1e-421a-909d-4e64447645b4 c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-fc20bd18-b90e-4028-a59e-82494c1cf687', 'ip', '-4', 'route', 'list', 'dev', 'tape8ade1ff-f6'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:04:55.047 27577 DEBUG neutron.agent.linux.utils [req-df7422fa-3f1e-421a-909d-4e64447645b4 c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-fc20bd18-b90e-4028-a59e-82494c1cf687', 'ip', '-6', 'route', 'list', 'dev', 'tape8ade1ff-f6'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:04:55.063 27577 DEBUG neutron.agent.linux.utils [req-df7422fa-3f1e-421a-909d-4e64447645b4 c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Unable to access /var/lib/neutron/external/pids/fc20bd18-b90e-4028-a59e-82494c1cf687.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:04:55.065 27577 DEBUG neutron.agent.linux.utils [req-df7422fa-3f1e-421a-909d-4e64447645b4 c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Unable to access /var/lib/neutron/external/pids/fc20bd18-b90e-4028-a59e-82494c1cf687.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:04:55.066 27577 DEBUG neutron.agent.linux.external_process [req-df7422fa-3f1e-421a-909d-4e64447645b4 c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] No process started for fc20bd18-b90e-4028-a59e-82494c1cf687 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:04:55.068 27577 DEBUG oslo_concurrency.lockutils [req-df7422fa-3f1e-421a-909d-4e64447645b4 c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Releasing semaphore "dhcp-agent-network-lock-fc20bd18-b90e-4028-a59e-82494c1cf687" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:04:58.649 27577 DEBUG oslo_concurrency.lockutils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Acquired semaphore "dhcp-agent-network-lock-fc20bd18-b90e-4028-a59e-82494c1cf687" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:04:58.650 27577 INFO neutron.agent.dhcp.agent [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T05:04:52Z, description=, device_id=50c3fd2e-0812-44c4-8b27-d8c497a7bd7e, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'90f939ae-63a5-46e2-b562-af2be2ccf784', u'ip_address': u'11.132.1.12'}], id=bffc516f-6853-4c5c-b6a9-05e09148c677, mac_address=fa:16:3e:27:10:1b, name=, network_id=fc20bd18-b90e-4028-a59e-82494c1cf687, port_security_enabled=True, project_id=0459e438b74d46959d6c89c9619846d9, revision_number=4, security_groups=[u'2b3abfdf-6a0f-454e-a746-e974e35a56de'], status=DOWN, tags=[], tenant_id=0459e438b74d46959d6c89c9619846d9, updated_at=2018-06-27T05:04:57Z
2018-06-27 07:04:58.652 27577 DEBUG neutron.agent.dhcp.agent [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Calling driver for network: fc20bd18-b90e-4028-a59e-82494c1cf687 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:04:58.653 27577 DEBUG neutron.agent.linux.dhcp [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Building host file: /var/lib/neutron/dhcp/fc20bd18-b90e-4028-a59e-82494c1cf687/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:04:58.658 27577 DEBUG neutron.agent.linux.dhcp [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Done building host file /var/lib/neutron/dhcp/fc20bd18-b90e-4028-a59e-82494c1cf687/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:04:58.665 27577 DEBUG neutron.agent.linux.utils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '31895'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:04:58.685 27577 DEBUG neutron.agent.linux.dhcp [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Reloading allocations for network: fc20bd18-b90e-4028-a59e-82494c1cf687 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:04:58.688 27577 DEBUG neutron.agent.linux.utils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-fc20bd18-b90e-4028-a59e-82494c1cf687', 'ip', '-4', 'route', 'list', 'dev', 'tape8ade1ff-f6'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:04:58.707 27577 DEBUG neutron.agent.linux.utils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-fc20bd18-b90e-4028-a59e-82494c1cf687', 'ip', '-6', 'route', 'list', 'dev', 'tape8ade1ff-f6'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:04:58.726 27577 DEBUG neutron.agent.linux.utils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Unable to access /var/lib/neutron/external/pids/fc20bd18-b90e-4028-a59e-82494c1cf687.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:04:58.728 27577 DEBUG neutron.agent.linux.utils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Unable to access /var/lib/neutron/external/pids/fc20bd18-b90e-4028-a59e-82494c1cf687.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:04:58.728 27577 DEBUG neutron.agent.linux.external_process [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] No process started for fc20bd18-b90e-4028-a59e-82494c1cf687 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:04:58.730 27577 DEBUG oslo_concurrency.lockutils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Releasing semaphore "dhcp-agent-network-lock-fc20bd18-b90e-4028-a59e-82494c1cf687" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:04:59.412 27577 DEBUG oslo_concurrency.lockutils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Acquired semaphore "dhcp-agent-network-lock-fc20bd18-b90e-4028-a59e-82494c1cf687" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:04:59.414 27577 INFO neutron.agent.dhcp.agent [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-27T05:04:52Z, description=, device_id=50c3fd2e-0812-44c4-8b27-d8c497a7bd7e, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'90f939ae-63a5-46e2-b562-af2be2ccf784', u'ip_address': u'11.132.1.12'}], id=bffc516f-6853-4c5c-b6a9-05e09148c677, mac_address=fa:16:3e:27:10:1b, name=, network_id=fc20bd18-b90e-4028-a59e-82494c1cf687, port_security_enabled=True, project_id=0459e438b74d46959d6c89c9619846d9, revision_number=5, security_groups=[u'2b3abfdf-6a0f-454e-a746-e974e35a56de'], status=DOWN, tags=[], tenant_id=0459e438b74d46959d6c89c9619846d9, updated_at=2018-06-27T05:04:58Z
2018-06-27 07:04:59.415 27577 DEBUG neutron.agent.dhcp.agent [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Calling driver for network: fc20bd18-b90e-4028-a59e-82494c1cf687 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:04:59.417 27577 DEBUG neutron.agent.linux.dhcp [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Building host file: /var/lib/neutron/dhcp/fc20bd18-b90e-4028-a59e-82494c1cf687/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:04:59.420 27577 DEBUG neutron.agent.linux.dhcp [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Done building host file /var/lib/neutron/dhcp/fc20bd18-b90e-4028-a59e-82494c1cf687/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:04:59.426 27577 DEBUG neutron.agent.linux.utils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '31895'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:04:59.447 27577 DEBUG neutron.agent.linux.dhcp [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Reloading allocations for network: fc20bd18-b90e-4028-a59e-82494c1cf687 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:04:59.450 27577 DEBUG neutron.agent.linux.utils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-fc20bd18-b90e-4028-a59e-82494c1cf687', 'ip', '-4', 'route', 'list', 'dev', 'tape8ade1ff-f6'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:04:59.468 27577 DEBUG neutron.agent.linux.utils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-fc20bd18-b90e-4028-a59e-82494c1cf687', 'ip', '-6', 'route', 'list', 'dev', 'tape8ade1ff-f6'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:04:59.486 27577 DEBUG neutron.agent.linux.utils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Unable to access /var/lib/neutron/external/pids/fc20bd18-b90e-4028-a59e-82494c1cf687.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:04:59.487 27577 DEBUG neutron.agent.linux.utils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Unable to access /var/lib/neutron/external/pids/fc20bd18-b90e-4028-a59e-82494c1cf687.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:04:59.488 27577 DEBUG neutron.agent.linux.external_process [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] No process started for fc20bd18-b90e-4028-a59e-82494c1cf687 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:04:59.489 27577 DEBUG oslo_concurrency.lockutils [req-421c88eb-a691-45f0-ae5e-8e58efaf89c3 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Releasing semaphore "dhcp-agent-network-lock-fc20bd18-b90e-4028-a59e-82494c1cf687" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:05:07.399 27577 DEBUG oslo_concurrency.lockutils [req-6d737020-7e6b-4ca5-966a-8b7b04ed000c c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Acquired semaphore "dhcp-agent-network-lock-fc20bd18-b90e-4028-a59e-82494c1cf687" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:05:07.401 27577 DEBUG neutron.agent.dhcp.agent [req-6d737020-7e6b-4ca5-966a-8b7b04ed000c c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Calling driver for network: fc20bd18-b90e-4028-a59e-82494c1cf687 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:05:07.404 27577 DEBUG neutron.agent.linux.utils [req-6d737020-7e6b-4ca5-966a-8b7b04ed000c c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-fc20bd18-b90e-4028-a59e-82494c1cf687', 'dhcp_release', 'tape8ade1ff-f6', '11.132.1.12', 'fa:16:3e:27:10:1b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:05:07.428 27577 DEBUG neutron.agent.linux.dhcp [req-6d737020-7e6b-4ca5-966a-8b7b04ed000c c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Building host file: /var/lib/neutron/dhcp/fc20bd18-b90e-4028-a59e-82494c1cf687/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:05:07.436 27577 DEBUG neutron.agent.linux.dhcp [req-6d737020-7e6b-4ca5-966a-8b7b04ed000c c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Done building host file /var/lib/neutron/dhcp/fc20bd18-b90e-4028-a59e-82494c1cf687/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:05:07.443 27577 DEBUG neutron.agent.linux.utils [req-6d737020-7e6b-4ca5-966a-8b7b04ed000c c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '31895'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:05:07.458 27577 DEBUG neutron.agent.linux.dhcp [req-6d737020-7e6b-4ca5-966a-8b7b04ed000c c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Reloading allocations for network: fc20bd18-b90e-4028-a59e-82494c1cf687 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:05:07.475 27577 DEBUG neutron.agent.linux.utils [req-6d737020-7e6b-4ca5-966a-8b7b04ed000c c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-fc20bd18-b90e-4028-a59e-82494c1cf687', 'ip', '-4', 'route', 'list', 'dev', 'tape8ade1ff-f6'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:05:07.491 27577 DEBUG neutron.agent.linux.utils [req-6d737020-7e6b-4ca5-966a-8b7b04ed000c c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-fc20bd18-b90e-4028-a59e-82494c1cf687', 'ip', '-6', 'route', 'list', 'dev', 'tape8ade1ff-f6'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:05:07.505 27577 DEBUG neutron.agent.linux.utils [req-6d737020-7e6b-4ca5-966a-8b7b04ed000c c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Unable to access /var/lib/neutron/external/pids/fc20bd18-b90e-4028-a59e-82494c1cf687.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:05:07.506 27577 DEBUG neutron.agent.linux.utils [req-6d737020-7e6b-4ca5-966a-8b7b04ed000c c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Unable to access /var/lib/neutron/external/pids/fc20bd18-b90e-4028-a59e-82494c1cf687.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:05:07.507 27577 DEBUG neutron.agent.linux.external_process [req-6d737020-7e6b-4ca5-966a-8b7b04ed000c c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] No process started for fc20bd18-b90e-4028-a59e-82494c1cf687 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:05:07.508 27577 DEBUG oslo_concurrency.lockutils [req-6d737020-7e6b-4ca5-966a-8b7b04ed000c c650e09f375e4378920843520bd0f1c0 0459e438b74d46959d6c89c9619846d9 - - -] Releasing semaphore "dhcp-agent-network-lock-fc20bd18-b90e-4028-a59e-82494c1cf687" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:05:24.615 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:05:24.619 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:06:24.615 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:06:24.619 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:07:16.017 27577 DEBUG oslo_concurrency.lockutils [req-e18bd3d7-30a7-4481-8822-a6384b160ce4 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:07:16.520 27577 DEBUG neutron.agent.dhcp.agent [req-7c62f23b-1c6e-4bc3-a207-c48b512a027f - - - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:07:16.523 27577 DEBUG neutron.agent.linux.utils [req-7c62f23b-1c6e-4bc3-a207-c48b512a027f - - - - -] Unable to access /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:07:16.525 27577 DEBUG neutron.agent.linux.dhcp [req-7c62f23b-1c6e-4bc3-a207-c48b512a027f - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc on network cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-06-27 07:07:16.526 27577 DEBUG neutron.agent.linux.dhcp [req-7c62f23b-1c6e-4bc3-a207-c48b512a027f - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc on network cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-06-27 07:07:19.475 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.495 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.512 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.537 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.553 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.574 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', 'link', 'set', 'tap7599dbf7-99', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.608 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-o', 'link', 'show', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.627 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-27 07:07:19.651 27577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap7599dbf7-99) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 07:07:19.653 27577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-06-27 07:07:19.655 27577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap7599dbf7-99) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 07:07:19.657 27577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'7599dbf7-9960-4cbb-bd7f-d6b0e3baa6dc', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:d2:f4:59'})), record=tap7599dbf7-99) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 07:07:19.693 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap7599dbf7-99', 'address', 'fa:16:3e:d2:f4:59'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.708 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.723 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap7599dbf7-99', 'netns', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.745 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', 'link', 'set', 'tap7599dbf7-99', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.760 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', 'link', 'set', 'tap7599dbf7-99', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.777 27577 DEBUG oslo_concurrency.lockutils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Acquired semaphore "iptables-qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:07:19.779 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.802 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.824 27577 DEBUG neutron.agent.linux.iptables_manager [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-06-27 07:07:19.826 27577 DEBUG oslo_concurrency.lockutils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Releasing semaphore "iptables-qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:07:19.827 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-o', 'link', 'show', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.843 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', 'addr', 'show', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.861 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'addr', 'add', '11.220.1.2/24', 'scope', 'global', 'dev', 'tap7599dbf7-99', 'brd', '11.220.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.883 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.903 27577 DEBUG neutron.agent.linux.dhcp [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Setting IPv4 gateway for dhcp netns on net cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc to 11.220.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1164
2018-06-27 07:07:19.907 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.220.1.1', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.925 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-6', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.944 27577 DEBUG neutron.agent.linux.dhcp [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Cleaning stale devices for network cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-06-27 07:07:19.946 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.961 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:19.988 27577 DEBUG neutron.agent.linux.dhcp [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:623
2018-06-27 07:07:19.991 27577 DEBUG neutron.agent.linux.dhcp [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/leases with contents:
1530162439 fa:16:3e:d2:f4:59 11.220.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:651
2018-06-27 07:07:19.992 27577 DEBUG neutron.agent.linux.dhcp [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Building host file: /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:07:19.994 27577 DEBUG neutron.agent.linux.dhcp [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Done building host file /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:07:19.998 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Unable to access /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:07:20.001 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host', '--addn-hosts=/var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap7599dbf7-99', '--dhcp-range=set:tag0,11.220.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:20.063 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:07:20.064 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:07:20.065 27577 DEBUG neutron.agent.linux.external_process [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:07:20.067 27577 DEBUG oslo_concurrency.lockutils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:07:20.069 27577 DEBUG oslo_concurrency.lockutils [req-e18bd3d7-30a7-4481-8822-a6384b160ce4 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:07:20.853 27577 DEBUG neutron.agent.dhcp.agent [req-98a2ad81-c086-4be0-af6c-f14a67084121 - - - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:07:20.856 27577 DEBUG neutron.agent.linux.dhcp [req-98a2ad81-c086-4be0-af6c-f14a67084121 - - - - -] Building host file: /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:07:20.859 27577 DEBUG neutron.agent.linux.dhcp [req-98a2ad81-c086-4be0-af6c-f14a67084121 - - - - -] Done building host file /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:07:20.870 27577 DEBUG neutron.agent.linux.utils [req-98a2ad81-c086-4be0-af6c-f14a67084121 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1098'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:20.895 27577 DEBUG neutron.agent.linux.dhcp [req-98a2ad81-c086-4be0-af6c-f14a67084121 - - - - -] Reloading allocations for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:07:20.896 27577 DEBUG neutron.agent.linux.utils [req-98a2ad81-c086-4be0-af6c-f14a67084121 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:20.915 27577 DEBUG neutron.agent.linux.utils [req-98a2ad81-c086-4be0-af6c-f14a67084121 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-6', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:20.933 27577 DEBUG neutron.agent.linux.utils [req-98a2ad81-c086-4be0-af6c-f14a67084121 - - - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:07:20.935 27577 DEBUG neutron.agent.linux.utils [req-98a2ad81-c086-4be0-af6c-f14a67084121 - - - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:07:20.936 27577 DEBUG neutron.agent.linux.external_process [req-98a2ad81-c086-4be0-af6c-f14a67084121 - - - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:07:20.937 27577 DEBUG oslo_concurrency.lockutils [req-98a2ad81-c086-4be0-af6c-f14a67084121 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:07:20.940 27577 DEBUG oslo_concurrency.lockutils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:07:20.941 27577 DEBUG neutron.agent.dhcp.agent [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T05:07:16Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'7036b3a7-0a5d-4ba1-8ff1-f0bb4d3c8492', u'ip_address': u'11.220.1.2'}], id=7599dbf7-9960-4cbb-bd7f-d6b0e3baa6dc, mac_address=fa:16:3e:d2:f4:59, name=, network_id=cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc, port_security_enabled=False, project_id=9abd4f2ddd424be2ac7a53239c7e8d6d, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=9abd4f2ddd424be2ac7a53239c7e8d6d, updated_at=2018-06-27T05:07:17Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:409
2018-06-27 07:07:20.942 27577 DEBUG oslo_concurrency.lockutils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:07:20.944 27577 DEBUG oslo_concurrency.lockutils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:07:20.946 27577 INFO neutron.agent.dhcp.agent [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-27T05:07:16Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'7036b3a7-0a5d-4ba1-8ff1-f0bb4d3c8492', u'ip_address': u'11.220.1.2'}], id=7599dbf7-9960-4cbb-bd7f-d6b0e3baa6dc, mac_address=fa:16:3e:d2:f4:59, name=, network_id=cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc, port_security_enabled=False, project_id=9abd4f2ddd424be2ac7a53239c7e8d6d, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=9abd4f2ddd424be2ac7a53239c7e8d6d, updated_at=2018-06-27T05:07:19Z
2018-06-27 07:07:20.949 27577 DEBUG neutron.agent.dhcp.agent [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:07:20.951 27577 DEBUG neutron.agent.linux.dhcp [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Building host file: /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:07:20.953 27577 DEBUG neutron.agent.linux.dhcp [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Done building host file /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:07:20.958 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1098'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:20.984 27577 DEBUG neutron.agent.linux.dhcp [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Reloading allocations for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:07:20.985 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:21.018 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-6', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:21.049 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:07:21.051 27577 DEBUG neutron.agent.linux.utils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:07:21.052 27577 DEBUG neutron.agent.linux.external_process [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:07:21.053 27577 DEBUG oslo_concurrency.lockutils [req-8efb03f4-78b5-45aa-a4e4-6f4259ccb797 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:07:24.616 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:07:24.620 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:07:56.926 27577 DEBUG oslo_concurrency.lockutils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:07:56.928 27577 INFO neutron.agent.dhcp.agent [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T05:07:55Z, description=, device_id=c92b36c8-7dec-4609-b246-8e9042b01267, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'7036b3a7-0a5d-4ba1-8ff1-f0bb4d3c8492', u'ip_address': u'11.220.1.1'}], id=4bcc487b-ec01-4853-b2c0-71bd9eafd9b7, mac_address=fa:16:3e:19:b7:f7, name=, network_id=cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc, port_security_enabled=False, project_id=9abd4f2ddd424be2ac7a53239c7e8d6d, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=9abd4f2ddd424be2ac7a53239c7e8d6d, updated_at=2018-06-27T05:07:56Z
2018-06-27 07:07:56.931 27577 DEBUG neutron.agent.dhcp.agent [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:07:56.934 27577 DEBUG neutron.agent.linux.dhcp [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Building host file: /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:07:56.936 27577 DEBUG neutron.agent.linux.dhcp [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Done building host file /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:07:56.945 27577 DEBUG neutron.agent.linux.utils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1098'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:56.969 27577 DEBUG neutron.agent.linux.dhcp [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Reloading allocations for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:07:56.971 27577 DEBUG neutron.agent.linux.utils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:56.992 27577 DEBUG neutron.agent.linux.utils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-6', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:57.009 27577 DEBUG neutron.agent.linux.utils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:07:57.018 27577 DEBUG neutron.agent.linux.utils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:07:57.019 27577 DEBUG neutron.agent.linux.external_process [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:07:57.020 27577 DEBUG oslo_concurrency.lockutils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:07:58.771 27577 DEBUG oslo_concurrency.lockutils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:07:58.772 27577 INFO neutron.agent.dhcp.agent [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T05:07:55Z, description=, device_id=c92b36c8-7dec-4609-b246-8e9042b01267, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'7036b3a7-0a5d-4ba1-8ff1-f0bb4d3c8492', u'ip_address': u'11.220.1.1'}], id=4bcc487b-ec01-4853-b2c0-71bd9eafd9b7, mac_address=fa:16:3e:19:b7:f7, name=, network_id=cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc, port_security_enabled=False, project_id=9abd4f2ddd424be2ac7a53239c7e8d6d, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=9abd4f2ddd424be2ac7a53239c7e8d6d, updated_at=2018-06-27T05:07:56Z
2018-06-27 07:07:58.773 27577 DEBUG neutron.agent.dhcp.agent [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:07:58.777 27577 DEBUG neutron.agent.linux.dhcp [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Building host file: /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:07:58.781 27577 DEBUG neutron.agent.linux.dhcp [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Done building host file /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:07:58.790 27577 DEBUG neutron.agent.linux.utils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1098'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:58.811 27577 DEBUG neutron.agent.linux.dhcp [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Reloading allocations for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:07:58.814 27577 DEBUG neutron.agent.linux.utils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:58.832 27577 DEBUG neutron.agent.linux.utils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-6', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:07:58.851 27577 DEBUG neutron.agent.linux.utils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:07:58.852 27577 DEBUG neutron.agent.linux.utils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:07:58.854 27577 DEBUG neutron.agent.linux.external_process [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:07:58.855 27577 DEBUG oslo_concurrency.lockutils [req-a34c2ac2-48a2-4c61-a91a-9a955bb4c8c7 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:08:02.671 27577 DEBUG oslo_concurrency.lockutils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:08:02.673 27577 INFO neutron.agent.dhcp.agent [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T05:07:55Z, description=, device_id=c92b36c8-7dec-4609-b246-8e9042b01267, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'7036b3a7-0a5d-4ba1-8ff1-f0bb4d3c8492', u'ip_address': u'11.220.1.1'}], id=4bcc487b-ec01-4853-b2c0-71bd9eafd9b7, mac_address=fa:16:3e:19:b7:f7, name=, network_id=cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc, port_security_enabled=False, project_id=9abd4f2ddd424be2ac7a53239c7e8d6d, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=9abd4f2ddd424be2ac7a53239c7e8d6d, updated_at=2018-06-27T05:07:57Z
2018-06-27 07:08:02.676 27577 DEBUG neutron.agent.dhcp.agent [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:08:02.679 27577 DEBUG neutron.agent.linux.dhcp [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Building host file: /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:08:02.682 27577 DEBUG neutron.agent.linux.dhcp [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Done building host file /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:08:02.690 27577 DEBUG neutron.agent.linux.utils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1098'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:02.706 27577 DEBUG neutron.agent.linux.dhcp [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Reloading allocations for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:08:02.708 27577 DEBUG neutron.agent.linux.utils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:02.724 27577 DEBUG neutron.agent.linux.utils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-6', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:02.738 27577 DEBUG neutron.agent.linux.utils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:08:02.739 27577 DEBUG neutron.agent.linux.utils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:08:02.741 27577 DEBUG neutron.agent.linux.external_process [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:08:02.742 27577 DEBUG oslo_concurrency.lockutils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:08:03.364 27577 DEBUG oslo_concurrency.lockutils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:08:03.365 27577 INFO neutron.agent.dhcp.agent [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-27T05:07:55Z, description=, device_id=c92b36c8-7dec-4609-b246-8e9042b01267, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'7036b3a7-0a5d-4ba1-8ff1-f0bb4d3c8492', u'ip_address': u'11.220.1.1'}], id=4bcc487b-ec01-4853-b2c0-71bd9eafd9b7, mac_address=fa:16:3e:19:b7:f7, name=, network_id=cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc, port_security_enabled=False, project_id=9abd4f2ddd424be2ac7a53239c7e8d6d, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=9abd4f2ddd424be2ac7a53239c7e8d6d, updated_at=2018-06-27T05:08:02Z
2018-06-27 07:08:03.366 27577 DEBUG neutron.agent.dhcp.agent [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:08:03.368 27577 DEBUG neutron.agent.linux.dhcp [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Building host file: /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:08:03.370 27577 DEBUG neutron.agent.linux.dhcp [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Done building host file /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:08:03.374 27577 DEBUG neutron.agent.linux.utils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1098'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:03.395 27577 DEBUG neutron.agent.linux.dhcp [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Reloading allocations for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:08:03.397 27577 DEBUG neutron.agent.linux.utils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:03.419 27577 DEBUG neutron.agent.linux.utils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-6', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:03.435 27577 DEBUG neutron.agent.linux.utils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:08:03.436 27577 DEBUG neutron.agent.linux.utils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:08:03.436 27577 DEBUG neutron.agent.linux.external_process [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:08:03.438 27577 DEBUG oslo_concurrency.lockutils [req-953ae4f7-6fd5-48e1-a97c-0da3c35789e9 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:08:11.779 27577 DEBUG oslo_concurrency.lockutils [req-de1d2b71-eec9-44a2-a7c2-b6bccb07ac8c 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Acquired semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:08:11.780 27577 DEBUG oslo_concurrency.lockutils [req-de1d2b71-eec9-44a2-a7c2-b6bccb07ac8c 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Releasing semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:08:15.937 27577 DEBUG oslo_concurrency.lockutils [req-a3937000-8e20-47db-87a6-a092aec4b280 - - - - -] Acquired semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:08:15.938 27577 DEBUG oslo_concurrency.lockutils [req-a3937000-8e20-47db-87a6-a092aec4b280 - - - - -] Releasing semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:08:16.728 27577 DEBUG oslo_concurrency.lockutils [req-a3937000-8e20-47db-87a6-a092aec4b280 - - - - -] Acquired semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:08:16.729 27577 DEBUG oslo_concurrency.lockutils [req-a3937000-8e20-47db-87a6-a092aec4b280 - - - - -] Releasing semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:08:24.616 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:08:24.620 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:08:36.698 27577 DEBUG oslo_concurrency.lockutils [req-99238551-0318-4037-b44a-941466b731d8 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:08:36.700 27577 INFO neutron.agent.dhcp.agent [req-99238551-0318-4037-b44a-941466b731d8 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T05:08:35Z, description=, device_id=8c468904-c0c8-4b42-8895-03d91625cf9b, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'7036b3a7-0a5d-4ba1-8ff1-f0bb4d3c8492', u'ip_address': u'11.220.1.7'}], id=e1d73604-fea3-4ac0-a9a7-1181958c9de7, mac_address=fa:16:3e:62:a7:df, name=, network_id=cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc, port_security_enabled=True, project_id=9abd4f2ddd424be2ac7a53239c7e8d6d, revision_number=3, security_groups=[u'61e0215a-ba60-4598-aaf1-9c2b1f3dc77d'], status=DOWN, tags=[], tenant_id=9abd4f2ddd424be2ac7a53239c7e8d6d, updated_at=2018-06-27T05:08:35Z
2018-06-27 07:08:36.702 27577 DEBUG neutron.agent.dhcp.agent [req-99238551-0318-4037-b44a-941466b731d8 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:08:36.706 27577 DEBUG neutron.agent.linux.dhcp [req-99238551-0318-4037-b44a-941466b731d8 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Building host file: /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:08:36.711 27577 DEBUG neutron.agent.linux.dhcp [req-99238551-0318-4037-b44a-941466b731d8 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Done building host file /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:08:36.720 27577 DEBUG neutron.agent.linux.utils [req-99238551-0318-4037-b44a-941466b731d8 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1098'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:36.744 27577 DEBUG neutron.agent.linux.dhcp [req-99238551-0318-4037-b44a-941466b731d8 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Reloading allocations for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:08:36.747 27577 DEBUG neutron.agent.linux.utils [req-99238551-0318-4037-b44a-941466b731d8 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:36.778 27577 DEBUG neutron.agent.linux.utils [req-99238551-0318-4037-b44a-941466b731d8 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-6', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:36.798 27577 DEBUG neutron.agent.linux.utils [req-99238551-0318-4037-b44a-941466b731d8 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:08:36.799 27577 DEBUG neutron.agent.linux.utils [req-99238551-0318-4037-b44a-941466b731d8 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:08:36.800 27577 DEBUG neutron.agent.linux.external_process [req-99238551-0318-4037-b44a-941466b731d8 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:08:36.801 27577 DEBUG oslo_concurrency.lockutils [req-99238551-0318-4037-b44a-941466b731d8 8f556dd5b6de4fef85ce3f9ee3d391f3 9abd4f2ddd424be2ac7a53239c7e8d6d - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:08:38.987 27577 DEBUG oslo_concurrency.lockutils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:08:38.989 27577 INFO neutron.agent.dhcp.agent [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T05:08:35Z, description=, device_id=8c468904-c0c8-4b42-8895-03d91625cf9b, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'7036b3a7-0a5d-4ba1-8ff1-f0bb4d3c8492', u'ip_address': u'11.220.1.7'}], id=e1d73604-fea3-4ac0-a9a7-1181958c9de7, mac_address=fa:16:3e:62:a7:df, name=, network_id=cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc, port_security_enabled=True, project_id=9abd4f2ddd424be2ac7a53239c7e8d6d, revision_number=4, security_groups=[u'61e0215a-ba60-4598-aaf1-9c2b1f3dc77d'], status=DOWN, tags=[], tenant_id=9abd4f2ddd424be2ac7a53239c7e8d6d, updated_at=2018-06-27T05:08:37Z
2018-06-27 07:08:38.991 27577 DEBUG neutron.agent.dhcp.agent [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:08:38.993 27577 DEBUG neutron.agent.linux.dhcp [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Building host file: /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:08:38.996 27577 DEBUG neutron.agent.linux.dhcp [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Done building host file /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:08:39.002 27577 DEBUG neutron.agent.linux.utils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1098'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:39.023 27577 DEBUG neutron.agent.linux.dhcp [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Reloading allocations for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:08:39.026 27577 DEBUG neutron.agent.linux.utils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:39.042 27577 DEBUG neutron.agent.linux.utils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-6', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:39.059 27577 DEBUG neutron.agent.linux.utils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:08:39.061 27577 DEBUG neutron.agent.linux.utils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:08:39.062 27577 DEBUG neutron.agent.linux.external_process [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:08:39.064 27577 DEBUG oslo_concurrency.lockutils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:08:39.828 27577 DEBUG oslo_concurrency.lockutils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:08:39.830 27577 INFO neutron.agent.dhcp.agent [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-27T05:08:35Z, description=, device_id=8c468904-c0c8-4b42-8895-03d91625cf9b, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'7036b3a7-0a5d-4ba1-8ff1-f0bb4d3c8492', u'ip_address': u'11.220.1.7'}], id=e1d73604-fea3-4ac0-a9a7-1181958c9de7, mac_address=fa:16:3e:62:a7:df, name=, network_id=cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc, port_security_enabled=True, project_id=9abd4f2ddd424be2ac7a53239c7e8d6d, revision_number=5, security_groups=[u'61e0215a-ba60-4598-aaf1-9c2b1f3dc77d'], status=DOWN, tags=[], tenant_id=9abd4f2ddd424be2ac7a53239c7e8d6d, updated_at=2018-06-27T05:08:39Z
2018-06-27 07:08:39.831 27577 DEBUG neutron.agent.dhcp.agent [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:08:39.833 27577 DEBUG neutron.agent.linux.dhcp [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Building host file: /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:08:39.835 27577 DEBUG neutron.agent.linux.dhcp [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Done building host file /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:08:39.854 27577 DEBUG neutron.agent.linux.utils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1098'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:39.875 27577 DEBUG neutron.agent.linux.dhcp [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Reloading allocations for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:08:39.879 27577 DEBUG neutron.agent.linux.utils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:39.905 27577 DEBUG neutron.agent.linux.utils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-6', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:08:39.929 27577 DEBUG neutron.agent.linux.utils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:08:39.930 27577 DEBUG neutron.agent.linux.utils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:08:39.932 27577 DEBUG neutron.agent.linux.external_process [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:08:39.933 27577 DEBUG oslo_concurrency.lockutils [req-204914f6-592e-4eaf-843e-df930b0c4643 db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:09:08.352 27577 DEBUG oslo_concurrency.lockutils [req-e40dd207-40cb-40d1-a3a7-a12f18c7c9fb 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:09:08.353 27577 DEBUG oslo_concurrency.lockutils [req-e40dd207-40cb-40d1-a3a7-a12f18c7c9fb 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Releasing semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:09:09.991 27577 DEBUG oslo_concurrency.lockutils [req-e40dd207-40cb-40d1-a3a7-a12f18c7c9fb 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:09:09.992 27577 DEBUG oslo_concurrency.lockutils [req-e40dd207-40cb-40d1-a3a7-a12f18c7c9fb 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Releasing semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:09:24.616 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:09:24.622 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.006s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:10:24.616 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:10:24.620 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:11:24.617 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:11:24.622 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:12:24.617 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:12:24.623 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:13:24.618 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:13:24.625 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.007s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:13:47.090 27577 DEBUG oslo_concurrency.lockutils [req-34f71c41-5e95-4452-a6d7-a7317c9a65ab 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:13:47.092 27577 DEBUG neutron.agent.dhcp.agent [req-34f71c41-5e95-4452-a6d7-a7317c9a65ab 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:13:47.095 27577 DEBUG neutron.agent.linux.utils [req-34f71c41-5e95-4452-a6d7-a7317c9a65ab 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'dhcp_release', 'tap7599dbf7-99', '11.220.1.7', 'fa:16:3e:62:a7:df'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:13:47.121 27577 DEBUG neutron.agent.linux.dhcp [req-34f71c41-5e95-4452-a6d7-a7317c9a65ab 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Building host file: /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:13:47.135 27577 DEBUG neutron.agent.linux.dhcp [req-34f71c41-5e95-4452-a6d7-a7317c9a65ab 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Done building host file /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:13:47.162 27577 DEBUG neutron.agent.linux.utils [req-34f71c41-5e95-4452-a6d7-a7317c9a65ab 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1098'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:13:47.179 27577 DEBUG neutron.agent.linux.dhcp [req-34f71c41-5e95-4452-a6d7-a7317c9a65ab 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Reloading allocations for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:13:47.181 27577 DEBUG neutron.agent.linux.utils [req-34f71c41-5e95-4452-a6d7-a7317c9a65ab 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:13:47.201 27577 DEBUG neutron.agent.linux.utils [req-34f71c41-5e95-4452-a6d7-a7317c9a65ab 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-6', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:13:47.221 27577 DEBUG neutron.agent.linux.utils [req-34f71c41-5e95-4452-a6d7-a7317c9a65ab 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:13:47.223 27577 DEBUG neutron.agent.linux.utils [req-34f71c41-5e95-4452-a6d7-a7317c9a65ab 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:13:47.224 27577 DEBUG neutron.agent.linux.external_process [req-34f71c41-5e95-4452-a6d7-a7317c9a65ab 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:13:47.225 27577 DEBUG oslo_concurrency.lockutils [req-34f71c41-5e95-4452-a6d7-a7317c9a65ab 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:14:24.619 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:14:24.623 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:14:43.064 27577 DEBUG oslo_concurrency.lockutils [req-0ff24daa-df45-4ef2-959a-ddfd9a4d0683 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:14:43.067 27577 DEBUG neutron.agent.dhcp.agent [req-0ff24daa-df45-4ef2-959a-ddfd9a4d0683 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:14:43.069 27577 DEBUG neutron.agent.linux.utils [req-0ff24daa-df45-4ef2-959a-ddfd9a4d0683 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'dhcp_release', 'tap7599dbf7-99', '11.220.1.1', 'fa:16:3e:19:b7:f7'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:14:43.096 27577 DEBUG neutron.agent.linux.dhcp [req-0ff24daa-df45-4ef2-959a-ddfd9a4d0683 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Building host file: /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 07:14:43.105 27577 DEBUG neutron.agent.linux.dhcp [req-0ff24daa-df45-4ef2-959a-ddfd9a4d0683 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Done building host file /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 07:14:43.118 27577 DEBUG neutron.agent.linux.utils [req-0ff24daa-df45-4ef2-959a-ddfd9a4d0683 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1098'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:14:43.134 27577 DEBUG neutron.agent.linux.dhcp [req-0ff24daa-df45-4ef2-959a-ddfd9a4d0683 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Reloading allocations for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 07:14:43.137 27577 DEBUG neutron.agent.linux.utils [req-0ff24daa-df45-4ef2-959a-ddfd9a4d0683 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-4', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:14:43.153 27577 DEBUG neutron.agent.linux.utils [req-0ff24daa-df45-4ef2-959a-ddfd9a4d0683 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc', 'ip', '-6', 'route', 'list', 'dev', 'tap7599dbf7-99'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:14:43.171 27577 DEBUG neutron.agent.linux.utils [req-0ff24daa-df45-4ef2-959a-ddfd9a4d0683 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:14:43.172 27577 DEBUG neutron.agent.linux.utils [req-0ff24daa-df45-4ef2-959a-ddfd9a4d0683 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:14:43.173 27577 DEBUG neutron.agent.linux.external_process [req-0ff24daa-df45-4ef2-959a-ddfd9a4d0683 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:14:43.174 27577 DEBUG oslo_concurrency.lockutils [req-0ff24daa-df45-4ef2-959a-ddfd9a4d0683 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:15:13.385 27577 DEBUG oslo_concurrency.lockutils [req-1ca061f7-7079-4e1b-ae45-3728241b7e5a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:15:13.386 27577 DEBUG neutron.agent.dhcp.agent [req-1ca061f7-7079-4e1b-ae45-3728241b7e5a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:15:13.390 27577 DEBUG neutron.agent.linux.utils [req-1ca061f7-7079-4e1b-ae45-3728241b7e5a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['kill', '-9', '1098'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:15:13.414 27577 DEBUG neutron.agent.linux.utils [req-1ca061f7-7079-4e1b-ae45-3728241b7e5a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-27 07:15:13.460 27577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tap7599dbf7-99) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 07:15:13.548 27577 DEBUG neutron.agent.linux.utils [req-378cc7c2-0297-41b6-8ddb-8e5d7f4bda6b - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:15:13.563 27577 DEBUG neutron.agent.linux.utils [req-378cc7c2-0297-41b6-8ddb-8e5d7f4bda6b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:15:13.577 27577 DEBUG oslo_concurrency.lockutils [req-378cc7c2-0297-41b6-8ddb-8e5d7f4bda6b - - - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:15:22.634 27577 DEBUG neutron.agent.dhcp.agent [req-94455b0c-88ca-4d77-a050-dd4d375add60 - - - - -] resync (cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:255
2018-06-27 07:15:22.636 27577 INFO neutron.agent.dhcp.agent [req-94455b0c-88ca-4d77-a050-dd4d375add60 - - - - -] Synchronizing state
2018-06-27 07:15:23.488 27577 INFO neutron.agent.dhcp.agent [req-439fc83b-e87b-4cbb-9a7a-9ea5b8a99640 - - - - -] All active networks have been fetched through RPC.
2018-06-27 07:15:23.490 27577 INFO neutron.agent.dhcp.agent [-] Starting network bccef923-8280-488c-84aa-955d7841d8ff dhcp configuration
2018-06-27 07:15:23.491 27577 INFO neutron.agent.dhcp.agent [-] Finished network bccef923-8280-488c-84aa-955d7841d8ff dhcp configuration
2018-06-27 07:15:23.492 27577 INFO neutron.agent.dhcp.agent [-] Starting network cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc dhcp configuration
2018-06-27 07:15:23.493 27577 DEBUG neutron.agent.dhcp.agent [-] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:15:23.495 27577 DEBUG neutron.agent.linux.utils [-] Unable to access /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:15:23.496 27577 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc on network cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-06-27 07:15:23.497 27577 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc on network cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-06-27 07:15:24.295 27577 DEBUG neutron.agent.linux.dhcp [req-fa07e32a-5539-4a0c-bc97-cd9befc210e9 - - - - -] Cleaning stale devices for network cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-06-27 07:15:24.296 27577 DEBUG neutron.agent.linux.utils [req-fa07e32a-5539-4a0c-bc97-cd9befc210e9 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:15:24.316 27577 DEBUG neutron.agent.dhcp.agent [req-fa07e32a-5539-4a0c-bc97-cd9befc210e9 - - - - -] Unable to enable dhcp for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc: there is a conflict with its current state; please check that the network and/or its subnet(s) still exist. call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:150
2018-06-27 07:15:24.317 27577 INFO neutron.agent.dhcp.agent [req-fa07e32a-5539-4a0c-bc97-cd9befc210e9 - - - - -] Finished network cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc dhcp configuration
2018-06-27 07:15:24.320 27577 INFO neutron.agent.dhcp.agent [req-439fc83b-e87b-4cbb-9a7a-9ea5b8a99640 - - - - -] Synchronizing state complete
2018-06-27 07:15:24.323 27577 DEBUG oslo_concurrency.lockutils [req-241b8a31-e205-437b-8940-0c7ce33193e3 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:15:24.620 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:15:24.623 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:15:25.099 27577 DEBUG neutron.agent.linux.utils [req-4357b0ff-f7b7-47f3-8df7-6cf90dcf970f - - - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:15:25.100 27577 DEBUG neutron.agent.linux.utils [req-4357b0ff-f7b7-47f3-8df7-6cf90dcf970f - - - - -] Unable to access /var/lib/neutron/external/pids/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:15:25.101 27577 DEBUG neutron.agent.linux.external_process [req-4357b0ff-f7b7-47f3-8df7-6cf90dcf970f - - - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:15:25.102 27577 DEBUG neutron.agent.dhcp.agent [req-4357b0ff-f7b7-47f3-8df7-6cf90dcf970f - - - - -] Calling driver for network: cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 07:15:25.103 27577 DEBUG neutron.agent.linux.utils [req-4357b0ff-f7b7-47f3-8df7-6cf90dcf970f - - - - -] Unable to access /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:15:25.104 27577 DEBUG neutron.agent.linux.utils [req-4357b0ff-f7b7-47f3-8df7-6cf90dcf970f - - - - -] Unable to access /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 07:15:25.105 27577 DEBUG neutron.agent.linux.external_process [req-4357b0ff-f7b7-47f3-8df7-6cf90dcf970f - - - - -] No process started for cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 07:15:25.106 27577 DEBUG neutron.agent.linux.dhcp [req-4357b0ff-f7b7-47f3-8df7-6cf90dcf970f - - - - -] Impossibile accedere a /var/lib/neutron/dhcp/cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:268
2018-06-27 07:15:25.107 27577 DEBUG neutron.agent.linux.dhcp [req-4357b0ff-f7b7-47f3-8df7-6cf90dcf970f - - - - -] No interface exists for network cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1482
2018-06-27 07:15:25.144 27577 DEBUG neutron.agent.linux.utils [req-88c53e0e-3037-4841-ac86-0c066eb82cb3 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 07:15:25.156 27577 DEBUG neutron.agent.linux.dhcp [req-88c53e0e-3037-4841-ac86-0c066eb82cb3 - - - - -] Namespace already deleted: qdhcp-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:246
2018-06-27 07:15:25.157 27577 DEBUG oslo_concurrency.lockutils [req-88c53e0e-3037-4841-ac86-0c066eb82cb3 - - - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:15:26.276 27577 DEBUG oslo_concurrency.lockutils [req-241b8a31-e205-437b-8940-0c7ce33193e3 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 07:15:26.277 27577 DEBUG oslo_concurrency.lockutils [req-241b8a31-e205-437b-8940-0c7ce33193e3 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Releasing semaphore "dhcp-agent-network-lock-cbd6b39d-2cba-4ecf-9cb8-96d3c32baecc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 07:16:24.620 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:16:24.624 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:17:24.620 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:17:24.624 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:18:24.620 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:18:24.625 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:19:24.621 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:19:24.625 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 07:20:24.621 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 07:20:24.623 27577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
